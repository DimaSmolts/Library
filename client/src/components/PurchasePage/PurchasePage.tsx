import React from 'react'
import PurchaseCompletionContainer from '../PurchaseCompletion'

export default class PurchasePage extends React.Component {
  render() {
    return <PurchaseCompletionContainer />
  }
}
