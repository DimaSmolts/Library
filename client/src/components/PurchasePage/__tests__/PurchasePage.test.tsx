import React from 'react'
import { shallow } from 'enzyme'

import PurchasePage from '../PurchasePage'

test('snapshot: <PurchasePage /> renders correctly', () => {
  const tree = shallow(
    <PurchasePage />)

  expect(tree).toMatchSnapshot()
})
