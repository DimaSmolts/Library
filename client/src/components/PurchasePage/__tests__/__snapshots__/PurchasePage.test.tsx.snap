// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot: <BookPage /> renders correctly 1`] = `<Connect(PurchaseCompletionContainer) />`;
